sw/device/silicon_creator/lib/ownership/BUILD

```diff
@@ -33,13 +33,19 @@ cc_library(
     name = "ecdsa",
     srcs = ["ecdsa.c"],
     hdrs = ["ecdsa.h"],
-    defines = ["USE_CRYPTOC=1"],
+    defines = [
+        "USE_OTBN=1",
+        #"USE_CRYPTOC=1",
```

Comment:
nit: are these two commented-out lines (here and a few lines below) left in intentionally? I see there are also some `#ifdefs` with `USE_CRYPTOC` still left in; will that still work, or should these be removed?

Reply:
They've been left in intentionally, as both implementations should be functional.

Once I get this merged and tested with the rest of the WIP ownership transfer code & tests, I'll eliminate CryptoC entirely.